docs/Kubernetes.md

```diff
@@ -5,17 +5,18 @@ Currently, the Calico network plugin for Kubernetes does not support ACL policie
 
 ## Getting Started
 First, you'll need to check out a copy of the Kubernetes git repo. Currently the Calico plugin is hosted in a Metaswitch repo, but will shortly be merged upstream.
-'''
+```
 git clone git@github.com:Metaswitch/calico-kubernetes.git
+cd calico_kubernetes
 git checkout calico-network-plugin-dev
-'''
+```
 
 Now set the environment variables to specify the Calico Vagrant provisioner, and run the cluster init script from the root of the kubernetes repo:
-'''
+```
 export KUBERNETES_PROVIDER=vagrant
 export NETWORK_MODE=calico
 cluster/kube-up.sh
-'''
+```
```

Comment:
Oops, this wasn't supposed to be here. Casey already implemented these changes and made a pull request to master.